spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+
+  describe "games#new action" do 
+    it "should require users to be logged in" do 
+      get :new
+      expect(response).to redirect_to new_user_session_path
+    end
+    
+    it "should return success response on the get request" do
+      user = create(:user)
+      sign_in user
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "games#show action" do 
+    it "should return success response on the get request" do
+      game = create(:game)
+      get :show, id: game.id
+      expect(response).to have_http_status(:success)
+    end
+
+    it "should return a Game Not Found error if the game is not found" do 
+      get :show, id: '99999999'
+      expect(response).to have_http_status(:not_found)
+    end
+  end
+
+  describe "games#update action" do 
+    before :each do 
+      @game = FactoryGirl.create(:game, name: "Test Name", state: "open")
+    end
+
+    context "with valid attributes" do
+      it "located the requested @game" do
+        user = create(:user)
+        sign_in user
+        put :update, id: @game, game: FactoryGirl.attributes_for(:game)
+        expect(@game).to eq(@game) 
```

Comment:
This is what I believe is a fairly standard approach to testing PUT methods. I understand limiting the number of tests and rolling the this into the next test, but I as far as locating the correct game before making any updates this seems correct to me.

http://everydayrails.com/2012/04/07/testing-series-rspec-controllers.html